README.md

```diff
@@ -1,56 +1,66 @@
-# Distributed version of the Spring PetClinic Sample Application built with Spring Cloud 
+# Distributed version of the Spring PetClinic Sample Application built with Spring Cloud
 
-[![Build Status](https://github.com/spring-petclinic/spring-petclinic-microservices/actions/workflows/maven-build.yml/badge.svg)](https://github.com/spring-petclinic/spring-petclinic-microservices/actions/workflows/maven-build.yml)
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![Build Status](https://github.com/clemlesne/azure-spring-apps-workshop-081122/actions/workflows/maven-build.yml/badge.svg)](https://github.com/clemlesne/azure-spring-apps-workshop-081122/actions/workflows/maven-build.yml)
 
 This microservices branch was initially derived from [AngularJS version](https://github.com/spring-petclinic/spring-petclinic-angular1) to demonstrate how to split sample Spring application into [microservices](http://www.martinfowler.com/articles/microservices.html).
-To achieve that goal, we use Spring Cloud Gateway, Spring Cloud Circuit Breaker, Spring Cloud Config, Spring Cloud Sleuth, Resilience4j, Micrometer 
-and the Eureka Service Discovery from the [Spring Cloud Netflix](https://github.com/spring-cloud/spring-cloud-netflix) technology stack.
+
+To achieve that goal, we use Spring Cloud Gateway, Spring Cloud Circuit Breaker, Spring Cloud Config, Spring Cloud Sleuth, Resilience4j, Micrometer and the Eureka Service Discovery from the [Spring Cloud Netflix](https://github.com/spring-cloud/spring-cloud-netflix) technology stack.
+
+# General usage
+
+## Prerequisites
+
+Be warning to use the right Java version (latest LTS, v17):
```

Comment:
Java 17 is the baseline of Spring Boot 3: https://spring.io/blog/2022/11/24/spring-boot-3-0-goes-ga
We have to upgrade. But for now I think we could support Java 11 and 17

Reply:
I didn't tested the build with both versions. I think it'll add unnecessary complexity to the code. It is really required to support a version which is not the latest LTS?

Reply:
Some Java developers are using this sample in their compagny where Java 17 may be not still available in live environment. It's quite commun in big non-IT organization. Thus keeping Java 11 compatibility could be comforting.